setup.py

```diff
@@ -2,10 +2,10 @@
 import os
 
 project_root = os.path.dirname(__file__)
-with open(os.path.join(project_root, "VERSION.txt")) as f:
+with open(os.path.join(project_root, "VERSION.txt"), "r", encoding="UTF-8") as f:
```

Comment:
Nit: Could you actually remove the `"r"` from both since it's the default mode anyway?